### DISCUSSION TITLE
Request for Refund of Accidental Pro Plan Charges

### DISCUSSION THREAD

**Discussion by NamjunKim12** [vercel/community#5828] [Account Issue]
### Summary

Dear Support Team,

I hope this message finds you well. I am writing to inform you about an issue I encountered while practicing distribution as a student studying development.

Regrettably, I accidentally created two pro plans, named "mangojelly" and "mangojelly1," resulting in a charge of $40. However, as there was a distribution failure, I have decided to delete both teams as they are no longer needed.

I kindly request your assistance in refunding the charged amount of $40. The invoice numbers associated with the accidental charges are as follows:

1. Team: "mangojelly"

   Invoice Number: D1F83742-0001

2. Team: "mangojelly1"

   Invoice Number: 07A158E0-0001

I apologize for any inconvenience caused and appreciate your prompt attention to this matter. Please let me know if any further information is required.

Thank you for your understanding and assistance.

Best regards,

Namjun

### Steps to Reproduce

Dear Support Team,

I hope this message finds you well. I am writing to inform you about an issue I encountered while practicing distribution as a student studying development.

Regrettably, I accidentally created two pro plans, named "mangojelly" and "mangojelly1," resulting in a charge of $40. However, as there was a distribution failure, I have decided to delete both teams as they are no longer needed.

I kindly request your assistance in refunding the charged amount of $40. The invoice numbers associated with the accidental charges are as follows:

1. Team: "mangojelly"

   Invoice Number: D1F83742-0001

2. Team: "mangojelly1"

   Invoice Number: 07A158E0-0001

I apologize for any inconvenience caused and appreciate your prompt attention to this matter. Please let me know if any further information is required.

Thank you for your understanding and assistance.

Best regards,

Namjun

**Comment by amyegan** [comment]
Hey @NamjunKim12. Please [submit a case](https://vercel.com/help) to the Customer Success Team. If you choose `Pricing Enquiry`, that will direct the case to our billing support team who can look into these invoices and let you know what refund options may be available.

**Reply by NamjunKim12** [reply]
I opened the case with the same content as this request yesterday, but I didn't get any answer. When can I get an answer to my inquiry?

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received. Please refrain from submitting multiple tickets for the same problem. You can find more info on response times at https://vercel.com/guides/vercel-support-queue-time

**Reply by NamjunKim12** [reply]
How long does it usually take for these refund inquiries to be processed? I was asked to contact biling@vercel.com again through my previous email reply, and I contacted that address, but there was no answer. As the Pro Plan is still in place, it needs to be processed quickly. I look forward to hearing from you.

**Reply by amyegan** [reply]
I don't have a specific time estimate since every case is different, but billing cases usually move fairly quickly. I'm happy to see that someone has already replied to your case. Please work with them to solve your billing issue and continue to follow up there with any further questions 🙏

**Reply by NamjunKim12** [reply]
Where can I look up the case I submitted? I submitted the case from a personal account related to this issue, but I don't know where I can find the submission.

**Reply by amyegan** [reply]
You should have an email notification where you can respond to update the case
